### PR TITLE
Create tabbar screen

### PR DESCRIPTION
**Pull request by framgia-nguyenkdm**

### CODE REVIEW

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
+//
+//  HomeTabBarController.swift
+//  Muzzic
+//
+//  Created by khuc.d.m.nguyen on 4/10/18.
+//  Copyright © 2018 khuc.d.m.nguyen. All rights reserved.
+//
+import UIKit
+enum HomeTabBarItem {
+    case home, playList
```
Comment (TranXuanThien) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: xuống dòng cho dễ nhìn em nhé

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+class TabBarItem: UITabBarItem {
+    var tabItem: HomeTabBarItem
+    init(item: HomeTabBarItem) {
+        self.tabItem = item
+        super.init()
+        self.image = item.image
+        self.selectedImage = item.selectedImage
+        self.title = item.title
+       // self.imageInsets = UIEdgeInsets(top: 6, left: 0, bottom: 0, right: 0)
```
Comment (daolq3012) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: remove comment code!

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+class HomeTabBarController: UITabBarController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configTabbar()
+        setSelectedItem(item: .home)
+    }
```
Comment (daolq3012) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: pls add a blank line in here!

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    func setSelectedItem(item: HomeTabBarItem) {
+        selectedIndex = item.index
+    }
```
Comment (daolq3012) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: pls add a blank line in here!

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
```
Comment (daolq3012) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: pls add a blank line in here!

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    func configTabbar() {
```
Comment (daolq3012) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: can be private

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift (hunk 3 above)
Comment (framgia-nguyenkdm) [reply Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: Done

File: Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+        switch self {
+        case .home:
+            return 0
+        case .playList:
+            return 1
+        }
+    }
+
+    static let tabbarItems = [HomeTabBarItem.home, HomeTabBarItem.playList]
+}
```
Comment (daolq3012) [inline Muzzic/Muzzic/ViewControllers/HomeTabBarController.swift]: pls add a blank line!

**Pull request merged**